### DISCUSSION TITLE
Is this for Godot4 or 3?

### DISCUSSION THREAD

**Discussion by robotsunknown** [niklas2902/py4godot#27] [Q&A] [answered]
I'm looking for any solution to call into Python from Godot4, and back.

**Comment by niklas2902** [comment, marked as answer]
Hi,
thank you, for your interest in this project. Yes, I'm currently working on moving this project to Godot 4 on my branch [godot4](https://github.com/niklas2902/py4godot/tree/godot4). However, I'm far from done and currently and I think, it will take until to summer, to get this to work.
